plugins/catalog-backend-module-github/config.d.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import { SchedulerServiceTaskScheduleDefinitionConfig } from '@backstage/backend-plugin-api';
+import { QueryOptions } from './src/lib/github';
```

Comment:
Unfortunately the config system reacts poorly to imported and reused types (yes the import above is a bit problematic as well). You may want to just paste the literal shape of the type in the four places below instead, and remove the import